db/migrate/20240606151348_create_ai_summaries_table.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class CreateAiSummariesTable < ActiveRecord::Migration[7.0]
+  def change
+    create_table :ai_summaries do |t|
+      t.integer :target_id, null: false
+      t.string :target_type, null: false
+      t.int4range :content_range
+      t.string :summarized_text, null: false
+      t.integer :meta_section_id
```

Comment:
We should drop this, and only migrate those where `meta_section_id` is null. This means they are the final summary shown to users. We never really use these small fragments so it's a good opportunity to get rid of them